apps/blog/README.md

```
@@ -1,83 +1,107 @@
 
-This repository contains the Nextjs version of the blog, which has been migrated from the old Gatsby version.
-The blog is now integrated with Auth0 for authentication and Vercel Storage KV for serverless Redis storage to support comments.
+Personal blog with MDX posts, KaTeX math rendering, and static pre-rendering.
 
-- **Live: https://blog.duyet.net** (official)
-- **Live: https://duyet.vercel.app** (Vercel)
-- **Live: https://duyet-blog.pages.dev** (Cloudflare Pages)
+- **Live**: https://blog.duyet.net (official)
+- **Live**: https://duyet-blog.pages.dev (Cloudflare Pages)
 
 ![](../../.github/screenshot/screenshot-blog.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)